scripts/synapse_port_db

```diff
@@ -947,6 +1037,7 @@ if __name__ == "__main__":
             postgres_config=postgres_config,
             progress=progress,
             batch_size=args.batch_size,
+            hs_config=config,
```

Comment:
I think that passing in the entire Config object makes the `postgres_config` param redundant, since it's available as `config.database_config`.